helix-view/Cargo.toml

```diff
@@ -29,3 +29,6 @@ slotmap = "1"
 serde = { version = "1.0", features = ["derive"] }
 toml = "0.5"
 log = "~0.4"
+
```

Comment:
Per caret requirements I avoided specifying the minor version so that we allow for major version bumps: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements